### PR TITLE
Feat validation

### PR DESCRIPTION
**Pull request by SpliiT**

### CODE REVIEW

File: blueprints/tools/blueprint_file_converter.py [hunk 1]
```diff
@@ -31,98 +30,92 @@ def file_converter_versions():
...
-    array_variables = ["filename"]
-    variables_dict = geode_functions.get_form_variables(
-        flask.request.form, array_variables
-    )
-    file_extension = os.path.splitext(variables_dict["filename"])[1][1:]
-    allowed_objects = geode_functions.list_objects(file_extension)
-
-    return flask.make_response({"allowed_objects": allowed_objects}, 200)
+    data = flask.request.json
+    filename = data.get("filename")
```
Comment (JulienChampagnol) [inline blueprints/tools/blueprint_file_converter.py:39]: By doing that you're remooving the automatic error gestion.
You should keep with the 'get_form_variables' and we should make some changes to it if necessary

File: blueprints/tools/blueprint_file_converter.py [hunk 2]
```diff
@@ -31,98 +30,92 @@ def file_converter_versions():
...
-
-    return response
\ No newline at end of file
+    geode_object = data.get("geode_object")
+    file = data.get("file")
+    filename = data.get("filename")
+    filesize = data.get("filesize")
+    extension = data.get("extension")
+
+    if geode_object and file and filename and filesize and extension:
```
Comment (JulienChampagnol) [inline blueprints/tools/blueprint_file_converter.py:72]: Same as above

File: blueprints/tools/blueprint_validity_checker.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def validity_checker_allowed_files():
 def validity_checker_allowed_objects():
     array_variables = ["filename"]
     variables_dict = geode_functions.get_form_variables(
-        flask.request.form, array_variables
+        flask.request.json, array_variables
```
Comment (JulienChampagnol) [inline blueprints/tools/blueprint_validity_checker.py:41]: Is it working here?
If so why not keep the same convension everywhere?

File: blueprints/workflows/blueprint_explicit.py [hunk 4]
```diff
@@ -93,10 +93,7 @@ def remesh():
     brep = geode_functions.load(
         "BRep", os.path.abspath(DATA_FOLDER + "explicit_brep.og_brep")
     )
-    try:
-        metric = float(variables["metric"])
-    except ValueError:
-        flask.abort(400, "Invalid data format for the metric variable")
+    flask.request.json["metric"]
```
Comment (JulienChampagnol) [inline blueprints/workflows/blueprint_explicit.py:96]: Why skipping the params validation here ?

File: blueprints/workflows/blueprint_implicit.py [hunk 5]
```diff
@@ -84,16 +84,9 @@ def update_value():
...
-    try:
-        point = int(variables["point"])
-    except ValueError:
-        flask.abort(400, "Invalid data format for the point")
-
-    try:
-        value = float(variables["value"])
-    except ValueError:
-        flask.abort(400, "Invalid data format for the value")
+    flask.request.json["point"]
```
Comment (JulienChampagnol) [inline blueprints/workflows/blueprint_implicit.py:87]: Same here

File: blueprints/tools/blueprint_file_converter.py
Comment (JulienChampagnol) [inline blueprints/tools/blueprint_file_converter.py:1]: In general make sure the unit tests are still working after the changes

**Review by JulienChampagnol — CHANGES_REQUESTED** [review]

File: blueprints/tools/blueprint_file_converter.py (hunk 1 above)
Comment (SpliiT) [reply blueprints/tools/blueprint_file_converter.py:39]: Ok !

**Pull request merged**